.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,12 +4,11 @@ relates to OR closes: #XXXX
 
 ---
 
-Before we can merge this PR, please make sure that all the following items have been
-checked off. If any of the checklist items are not applicable, please leave them but
-write a little note why.
-
+Before we can approve this PR for merge, please make sure that **all** the following items have been checked off:
 - [ ] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
-- [ ] Added **one** line describing your change in `<branch>/CHANGELOG.md`
+- [ ] Added *exactly* one label from the `L-` group against this PR
+- [ ] Added *at least* one label from the `A-` and one label from the `C-` group against either this PR or its associated issue
+- [ ] Either this PR or its associated issue is targeted against the *current* Milestone ( otherwise discuss if it can be added )
```

Comment:
I don't think putting the milestone on the PR will show it on the milestone board, unless it also has a Project. So maybe mention that as well. But these special cases would just make things more confusing imo.
I'd still prefer to only have issues on the milestone board with linked PRs.

Reply:
Yeah, i agree - I'm looking for ideas to design this in a way that

a) avoids duplication of info
b) contains all the needed info for the CHANGELOG in the PR
c) handles the special cases where a PR does not close an issue
d) has no ambiguity, no `either or`s or `if`s

It's possible that all these at the same time are not possible, in which case i'd be willing to drop a) if we can keep d)

Reply:
i just saw we can filter the milestone list with `is:issue` and `is:PR` for planning and merge-date sync respectively.

Given that, I think we should be adding both PRs and Issues to the project
![image](https://user-images.githubusercontent.com/4237655/183261537-fd7de21e-4a85-474b-9fe7-1c93268b4ce0.png)

Reply:
Currently, we don't distinguish which PR is for calamari or which PR is for dolphin, which is for client or runtime.
Like polkadot, there're prefixes like `P` means `Polkadot`, `S` means `Substrate`

Reply:
with the PRs we have the `A-calamari` `A-manta` and `A-dolphin`

we could apply them to issues as well